enterprise/backend/src/metabase_enterprise/audit_app/pages/common/cards.clj

```diff
@@ -25,6 +25,22 @@
                        :group-by [:card_id]}
                       (common/add-45-days-clause :started_at))])
 
+(def latest-qe
+  "HoneySQL for a CTE to get latest QueryExecution for a Card."
+  [:latest_qe {:select [:query_execution.card_id :error :query_execution.started_at]
+               :from [:query_execution]
+               :join [[{:select [:card_id [:%max.started_at :started_at]]
+                       :from [:query_execution]
+                       :group-by [:card_id]} :inner_qe]
+                      [:= :query_execution.started_at :inner_qe.started_at]]}])
```

Comment:
IIRC CTEs don't work on MySQL 5.7 which is our minimum supported version. You can rewrite this as a subselect tho

Reply:
Nevermind I forgot we have the code that rewrites CTEs to subselects for MySQL automatically in the audit app

Reply:
huh? we have cte's on the existing `table` query though?